docs/user-guide/class-b-beaconing.md

```diff
@@ -0,0 +1,115 @@
+# Class-B: Beacon locking
```

Comment:
We should also add a section on the Concentrator provisioning section to explain which twins should be set for Class B devices.

Reply:
I don't know what you mean.

Reply:
Sorry I should have been more specific. We should document on this page what are the optional device twins needed to activate the Class B beaconing : https://azure.github.io/iotedge-lorawan-starterkit/2.2.0/user-guide/station-device-provisioning/#manual-configuration

Reply:
I opened an issue for it: https://github.com/Azure/iotedge-lorawan-starterkit/issues/1959